src/useScrollLinking/useScrollLinking.ts

```diff
@@ -0,0 +1,64 @@
+import { onMounted, onUnmounted, ref, Ref } from 'vue'
+import { MaybeRef } from '@/types/maybe'
+
+type DisconnectScrollLink = () => void
+type ScrollLinkReturnValue = {
```

Comment:
nit: This repo is pretty consistent with using the name of the composition as the return type. So this would be `UseScrollLinking`